src/Razor/src/Microsoft.NET.Sdk.Razor/Microsoft.NET.Sdk.Razor.csproj

```diff
@@ -36,6 +36,8 @@
     <Compile Include="..\Microsoft.AspNetCore.Razor.Tools\Client.cs">
       <Link>Shared\Client.cs</Link>
     </Compile>
+
+    <Content Include="_._" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
Maybe I should add a reference to https://github.com/NuGet/NuGet.Client/pull/2897